monitor/internal/pod/delete_controller.go

```diff
@@ -31,10 +33,17 @@ type DeleteController struct {
 	reconcileFunc      deleteControllerReconcileFunc
 	tickerPeriod       time.Duration
 	itemProcessTimeout time.Duration
+	sandboxExtractor   extractors.PodSandboxExtractor
+}
+
+type DeleteObject struct {
+	nativeID string
```

Comment:
you are already using the nativeID as key ... not really necessary to have it in here as well, no?

Reply:
Yup, good point. WIll make changes.